.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release Phar
+
+on: release
```

Comment:
Maybe I'm not following properly?

You select the tag to be created when making a release and I want to be able to define exactly what goes in the release notes, so my hope with this is that I can still do that, but now I no longer have to manually build the phar locally and manually upload it after as I have been doing.

![image](https://user-images.githubusercontent.com/1066195/154586987-094a7faa-0e77-4d07-a8ca-f9b78d3987b5.png)

Reply:
Makes sense. I just don't like having to manually create a release and would simply reference change-log docs. 🤷

Reply:
I often have to include some instructions or caveats on releases and since `so` links to the release itself when an update is available, it's right there for the user to see what's new etc...